Code2pdf/code2pdf.py

```diff
@@ -142,9 +147,19 @@ def parse_arg():
 
 def main():
     args = parse_arg()
-    pdf_file = get_output_file(args.filename, args.outputfile)
-    pdf = Code2pdf(args.filename, pdf_file, args.size)
-    pdf.init_print(linenos=args.linenos, style=args.style)
+    if  (args.filelist == False):
+        pdf_file = get_output_file(args.filename, args.outputfile)
+        pdf = Code2pdf(args.filename, pdf_file, args.size)
+        pdf.init_print(linenos=args.linenos, style=args.style)
+    else:
+        with open(args.filename) as filelist:
```

Comment:
@cibinjoseph I doubt here `with open` would work this way. It expects a string or buffer and not a list. Even if type of `filename` is not a list but a string, you may still want to split the filenames and feed to `open` separately.

Anyway, that's why I asked for a test suite. :)

Reply:
It seems to work for me. 
`open(args.filename) as filelist:` opens the file that **contains the list of file names**. 
The next line `filenames=filelist.read().splitlines()` would parse `filelist` for the names of the files and then store it as a list in `filenames`.

And yes, I did want to finish the test suite but got caught up with some work. Don't accept without the test suite! :D

Reply:
I remember getting a segmentation fault from PyQT after the program run (for the copy in your repo too). That seemed to deter me from working on the test suite. Have to check if it still exists when PyQT is compiled from scratch.

Reply:
@cibinjoseph Cool! I misunderstood `filename` being comma separated list of file names passed as a console arg. :)

Please feel free to look into the seg fault issue as I've not been able to investigate myself due to work commitments. I believe there is already an [issue](https://github.com/tushar-rishav/code2pdf/issues/20) created by you for this.
Also, to be consistent, I will go ahead with your current PR once you've added the suite. Thanks!